index.html

```diff
@@ -134,6 +143,36 @@ <h1 class="works-project-title">Profesional Art Printing Data</h1>
         </div>
         <button class="works-project-button">See Project</button>
       </li>
+      <li class="works-project">
+        <h1 class="works-project-title">Data Dashboard Healthcare</h1>
+        <p class="works-project-content">
+          A daily selection of privately personalized reads; no accounts or sign-ups
+          required. has been the industry's standard
+        </p>
+        <div class="works-project-languages">
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="works-project-button">See Project</button>
+      </li>
+      <li class="works-project">
+        <h1 class="works-project-title">Website Protfolio</h1>
+        <p class="works-project-content">
+          A daily selection of privately personalized reads; no accounts or sign-ups
+          required. has been the industry's standard
+        </p>
+        <div class="works-project-languages">
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="works-project-button">See Project</button>
+      </li>
     </ul>
   </section>
```

Comment:
Kindly modify the project section to match that of the design. You can observe the margins and also observe that the project images are portrait-like and not landscape.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187029268-9a15419c-b21d-428a-9a65-9804d75a5caa.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029292-26597039-d842-4c15-b924-79af54840527.png)